### PR TITLE
ICDS Dashboard: remove info button

### PR DESCRIPTION
**Pull request by saiprakash-v**
Trello link: https://trello.com/c/IeFcLR3r
##### SUMMARY

Hiding info button in rankings page

### CODE REVIEW

**Review by czue — APPROVED** [review]

File: custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html [hunk 1]
```diff
@@ -22,9 +22,12 @@
           <span class="font-16 font-montserrat font-weight-600 mr-3">
             {$ $ctrl.location_type === 'Awc' ? $ctrl.location_type.toUpperCase() : $ctrl.location_type $}s Ranking
           </span>
+          {% comment %}
+          Hiding info button
```
Comment (czue) [inline custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html]: Any reason not to just delete this?

Comment (saiprakash-v) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html]: We can delete it. Just in case if they might want it back in future

Comment (czue) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html]: yeah let's just delete it, easy enough to add back

Comment (saiprakash-v) [reply custom/icds_reports/templates/icds_reports/icds_app/mobile/map-chart.html]: done

**Review by czue — APPROVED** [review]

**Pull request merged**